lib/api/utilities/endpoints/bodied.rb

```diff
@@ -91,6 +104,7 @@ def process(current_user, instance, params)
 
           process_service
             .new(**args.compact)
+            .with_state(process_state.call(model: instance, params: params))
```

Comment:
I have been wondering for some time now whether it would make sense to make the params available within the state of a service. That way, we would not have to pass the params around and could also access them in the callbacks which could then more easily be turned into default `ActiveModel::Callbacks`.

Do you have an opinion on this, @oliverguenther ?

Reply:
Yes that sounds like a good convention.